packages/puppeteer-core/src/common/bidi/Frame.ts

```diff
@@ -103,6 +109,31 @@ export class Frame extends BaseFrame {
     return this.#page.frame(this._parentId ?? '');
   }
 
+  /**
+   * Gets the frame element associated with this frame. If the frame is
+   * top-level, then this returns nothing.
+   */
+  async frameElement(): Promise<HandleFor<HTMLIFrameElement> | undefined> {
```

Comment:
Let's discuss if we want to use `undefined` or `null`. Both are OK but would prefer all our API's to be consistent.

Reply:
This is internal.